test/unit/tasks/migrate_alt_text_to_caption_test.rb

```diff
@@ -0,0 +1,87 @@
+require "test_helper"
+
+class MigrateAltTextToCaptionTest < ActiveSupport::TestCase
+  setup do
+    @task = Rake::Task["data_migration:alt_text_to_caption"]
+    @task.reenable
+  end
+
+  test "migration task is idempotent" do
```

Comment:
This test doesn't quite do what I'd expect. I'd sooner it checks that if the first run only processes a proportion of the alt texts, then a second run will continue churning through the unprocessed ones.

I'd suggest doing that by creating a new image after the first run, and checking that the second run catches it. And also, ideally, checking that the second run doesn't call `update`/`update_column` / whatever on the images that were already processed in the first run.